app/javascript/dashboard/i18n/locale/en/chatlist.json

```diff
@@ -80,6 +80,7 @@
     "NO_MESSAGES": "No Messages",
     "NO_CONTENT": "No content available",
     "HIDE_QUOTED_TEXT": "Hide Quoted Text",
-    "SHOW_QUOTED_TEXT": "Show Quoted Text"
+    "SHOW_QUOTED_TEXT": "Show Quoted Text",
+    "NEW_CONVERSATION_CREATED": "New conversations available to reload"
```

Comment:
New conversations are available. Click here to refresh.

Reply:
Is it good to show the text "Click here to refresh." as a button instead of showing reload icon.

Which is good? @pranavrajs @nithindavid 

1.
<img width="796" alt="Screenshot 2022-02-08 at 10 29 13 AM" src="https://user-images.githubusercontent.com/64252451/152921431-b2f7f15f-4c85-4a13-bd53-4808b1337cad.png">

2.
<img width="796" alt="Screenshot 2022-02-08 at 10 35 58 AM" src="https://user-images.githubusercontent.com/64252451/152921778-97c94010-075b-479c-bf4d-f421979167b7.png">